### DISCUSSION TITLE
OSRM för isokroner

### DISCUSSION THREAD

**Discussion by soleildeminuit** [salgo60/Svenskabadplatser#88] [Ideas]
Open Source Routing Machine (OSRM) bygger på Open Street Map och har stöd för bil, cykel och till fots för isokroner ("hur långt kommer man på x minuter?"). Exempel Vesljungasjön.
![isokron](https://user-images.githubusercontent.com/10654578/119258713-3660c980-bbcb-11eb-86a6-ebff860831ae.PNG)

**Comment by salgo60** [comment]
Tackar skrev detta på [FB](https://www.facebook.com/groups/vindskyddinorden/permalink/1416472078701691/?comment_id=1416504338698465&reply_comment_id=1416831371999095) om just hur man skall ange för ett vindskydd var det är så att koppling kommunala transporter skall fungera

![image](https://user-images.githubusercontent.com/14206509/119261649-a6c21780-bbd8-11eb-8500-455c93ebf14a.png)

**Reply by salgo60** [reply]
* [Facebook](https://www.facebook.com/groups/vindskyddinorden/permalink/1417966058552293/?comment_id=1417989151883317) 

![image](https://user-images.githubusercontent.com/14206509/119476323-3dfcab80-bd4e-11eb-8eca-07f89c85ebf0.png)

**Reply by salgo60** [reply]
SVT har skapa en del kul grejer
*  ["Coronasommaren 2020: Så långt kommer du på två timmar"](https://www.svt.se/datajournalistik/sa-langt-kommer-du-pa-tva-timmar/)
![image](https://user-images.githubusercontent.com/14206509/119477678-7355c900-bd4f-11eb-9572-f580d0d7dc1f.png)
* [Var finns närmaste skyddsrum?](https://www.svt.se/special/sveriges-skyddsrum/)
![image](https://user-images.githubusercontent.com/14206509/119477903-a9934880-bd4f-11eb-8384-ca5ca6bd313a.png)

**Comment by salgo60** [comment]
länk från [pangoSE](https://github.com/pangoSE)
* [mobilitydata.org](https://mobilitydata.org/)
![image](https://user-images.githubusercontent.com/14206509/119702974-5b1ca180-be56-11eb-9e43-ea83f74fd698.png)

  * [gtfs.mobilitydata.org/about](https://gtfs.mobilitydata.org/about)
![image](https://user-images.githubusercontent.com/14206509/119702495-ca45c600-be55-11eb-8aa5-2e12d86fb689.png)

* [How to adapt your GTFS dataset to COVID-19](https://docs.google.com/document/d/1qLcagdhXZthjRL7C5zz1JOzgkpGAPsxDu58RpprP9pc/edit
)
![image](https://user-images.githubusercontent.com/14206509/119703958-63291100-be57-11eb-85dd-f3eba228f78e.png)
